src/devices/lcec_epocat.c

```diff
@@ -615,110 +703,135 @@ static void lcec_epocat_write(struct lcec_slave *slave, long period) {
   EC_WRITE_BIT(&pd[hal_data->off_ENABLE_5V], 3, *(hal_data->ENABLE_5V_3));
   EC_WRITE_BIT(&pd[hal_data->off_ENABLE_5V], 4, *(hal_data->ENABLE_5V_4));
 
-  fValue = (((*(hal_data->dac_00_value) * 1000) / hal_data->enc_00_scale) / 16) * hal_data->dac_00_scale;
-  if (fValue > 65534) {
-    fValue = 65534;
+  fValue = (((*(hal_data->dac_00_value) * 1000) / hal_data->enc_00_scale)) * hal_data->dac_00_scale;
```

Comment:
Consider pulling this out into its own function, as it's repeated a few times.  Maybe take (`dac_xx_value`, `enc_xx_scale`, `dac_xx_scale`) and return fValue?  Or, alternately, pass in an offset as well and let it do the writes.

Reply:
Yes, You are absolutely right. In my code, there are a lot of parts repeated; I did it fast to test it, but now it seems better (for me) to read if there are the full lines, but it is only my opinion. I always coded only for myself; this is the first time that my code is public :-), so it is now difficult for me to adapt to this :-). Also, it's difficult for me to understand how GitHub works.

Reply:
This morning I changed the epocat.h file (without a pull request). I would to restore the version of yesterday and I don't know how to do it (via web).

Reply:
Let's do this: I'll go ahead and merge this PR and then make a few quick edits on my end.  Then feel free to open up additional pull requests for whatever is missing/left.  Ok?